### PR TITLE
Add Project Status in README

### PR DESCRIPTION
**Pull request by trivikr**
Fixes: https://github.com/open-telemetry/opentelemetry-java/issues/4611

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 [![Coverage Status][codecov-image]][codecov-url]
 [![Maven Central][maven-image]][maven-url]
 
+## Project Status
+
+Please refer to [OpenTelemetry Instrumentation for Java][otel-instrumentation-java].
```
Comment (chalin) [inline README.md:8]:
```suggestion
See [OpenTelemetry Instrumentation for Java][otel-instrumentation-java].
```

File: README.md [hunk 2]
```diff
@@ -326,3 +330,4 @@ Made with [contrib.rocks](https://contrib.rocks).
 [opentelemetry-java-instrumentation]: https://github.com/open-telemetry/opentelemetry-java-instrumentation
 [opentelemetry-java-docs]: https://github.com/open-telemetry/opentelemetry-java-docs
 [opentelemetry.io]: https://opentelemetry.io
+[otel-instrumentation-java]: https://opentelemetry.io/docs/instrumentation/java/
```
Comment (chalin) [inline README.md:333]:
```suggestion
[otel-instrumentation-java]: https://opentelemetry.io/docs/instrumentation/java/#status-and-releases
```

**Review by chalin — COMMENTED** [review]
See inline suggested comments. Thanks! /cc @austinlparker @cartermp

**Review by jkwatson — APPROVED** [review]

**Review by chalin — APPROVED** [review]
👍

**Pull request merged**